mmv1/third_party/terraform/tests/resource_data_loss_prevention_deidentify_template_test.go

```diff
@@ -401,3 +401,191 @@ resource "google_kms_key_ring" "key_ring" {
+        fields {
+          name = "details.pii.email"
+        }
+        condition {
+          expressions {
+            conditions {
+              conditions {
+                field {
+                  name = "details.pii.date_of_birth"
+                }
+                operator = "GREATER_THAN_OR_EQUALS"
+                value {
+                  date_value {
+                    year = 2001
+                    month = 6
+                    day = 29
+                  }
+                }
+              }
+            }
+          }
+        }
+        primitive_transformation {
+          replace_config {
+            new_value {
+              string_value = "born.after.shrek@example.com"
+            }
+          }
+        }
+      }
+      field_transformations {
+        fields {
+          name = "unconditionally-redacted-field"
+        }
+        primitive_transformation {
+          redact_config {}
+        }
+      }
+      field_transformations {
+        fields {
+          name = "unconditionally-char-masked-field"
+        }
+        primitive_transformation {
+          character_mask_config {
+            masking_character = "x"
+            number_to_mask = 8
+            characters_to_ignore {
+              characters_to_skip = "-"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+`, context)
+}
+
+func testAccDataLossPreventionDeidentifyTemplate_dlpDeidentifyTemplate_recordTransformations_update(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_data_loss_prevention_deidentify_template" "basic" {
+  parent = "organizations/%{organization}"
+  description = "Description"
+  display_name = "Displayname"
+
+  deidentify_config {
+    record_transformations {
+      field_transformations {
+        fields {
+          name = "details.pii.email"
+        }
+        condition {
+          expressions {
+            conditions {
+              # update to add second condition
+              conditions {
+                field {
+                  name = "details.pii.gender"
+                }
+                operator = "EQUAL_TO"
+                value {
+                  string_value = "M"
+                }
+              }
+              conditions {
+                field {
+                  name = "details.pii.date_of_birth"
+                }
+                operator = "GREATER_THAN_OR_EQUALS"
+                value {
+                  # update date values
+                  date_value {
+                    year = 2004
+                    month = 7
+                    day = 2
+                  }
+                }
+              }
+            }
+          }
+        }
+        primitive_transformation {
+          # update values inside replace_config
+          replace_config {
+            new_value {
+              string_value = "dude.born.after.shrek2@example.com"
+            }
+          }
+        }
+      }
+      field_transformations {
```

Comment:
could be worth removing this field transformation to make sure removal-on-update works as expected.

Reply:
Ditto what I said for condition blocks